backend/handler/database/roms_handler.py

```diff
@@ -122,6 +122,17 @@ def update_rom(self, id: int, data: dict, session: Session = None) -> Rom:
             .execution_options(synchronize_session="evaluate")
         )
 
+    @begin_session
+    def set_main_sibling(self, rom: Rom, user_id: int, session: Session = None) -> None:
+        siblings = [rom.id for rom in rom.get_sibling_roms()]
+        return session.execute(
+            update(UserRomProps)
+            .where(
+                and_(UserRomProps.rom_id.in_(siblings), UserRomProps.user_id == user_id)
+            )
+            .values(is_main_sibling=False)
+        )
```

Comment:
nothing in here sets the main sibling to true?

Reply:
the asignation to true is in a previous query in the endpoint function, but i need to merge both queries to make them atomic